### ISSUE TITLE
Support for fira code font?

### ISSUE THREAD

**Issue by tifDev** [gnunn1/tilix#1891]
Is it possible to set the terminal font to `fira code` ?

**Comment by mdhenriksen** [comment]
Yes. Just to to preferences -> Choose your profile -> General -> Check "custom font" -> Set to Fira Code. Should do the trick, I use it in mine.

**Comment by tifDev** [comment]
I don't know why this appears so horrible on my distro (i tried with other distros and i don't have this problem)
[img](https://i.ibb.co/R6Ks2B0/image.png)

**Comment by mdhenriksen** [comment]
> I don't know why this appears so horrible on ZorinOS distro (i tried with other distros and i don't have this problem)
> [img](https://i.ibb.co/R6Ks2B0/image.png)

Try the Nerd Fonts version of Fira Code, that's the only thing I can think of might work - It's optimized for terminal use as far as I know. You can download it here: https://www.nerdfonts.com/font-downloads
